import_ldraw.py

```diff
@@ -298,18 +306,18 @@ def parse(self, filename):
                     if tmpdate != "":
                         tmpdate = tmpdate.split()
 
-                        # TODO What is this condition for?
-                        # le717 unable to find a case where it is hit.
-                        if (tmpdate[0] == "0" and
-                            len(tmpdate) >= 3 and
-                            tmpdate[1].lower() == "!ldraw_org" and
-                            "part" in tmpdate[2].lower() and
-                            self.part_count > 1
-                        ):
-                            self.subparts.append(
-                                [filename, self.mat, self.colour]
-                            )
-                            break
+                        # LDraw brick comments
```

Comment:
Ah, OK. That makes sense. This seems to go back to `isSubPart()` needing correcting. That should really be one of the next big changes made.